backends/graphics/sdl/sdl-graphics.cpp

```diff
@@ -336,6 +347,12 @@ bool SdlGraphicsManager::createOrUpdateWindow(int width, int height, const Uint3
 	if (!_window) {
 		return false;
 	}
+	Uint32 windowFlags = flags;
+	if (_lockedScreen) {
+		windowFlags &= ~SDL_WINDOW_RESIZABLE;
+	} else {
+		windowFlags |= SDL_WINDOW_RESIZABLE;
+	}
```

Comment:
Since `windowFlags` is not used anywhere, this code doesn't affect any functionality